### DISCUSSION TITLE
skipped titles keep coming back.

### DISCUSSION THREAD

**Discussion by Dave-Kay** [rern/rAudio#867] [Bug Reports]
on my list are a few hundred tracks, i like to shuffle them over all.
but if there is a track i don't really want to listen to right now, i skip it, but, it will be next on list again....
was there in first release, ist still there in the last release i installed a few days ago and updated yesterday.

**Comment by cYCL157** [comment]
i have this same issue and understood it to be an issue with mpd, but would be nice to have really skip.

**Comment by EarsofTin** [comment]
Would this be a workaround?  Add your playlist to the queue, turn shuffle on, turn consume mode on.  When you skip a track in consume mode it is deleted from the queue as is each track when it completes.  When the queue runs out, repopulate from your saved playlists or file directories.

**Reply by cYCL157** [reply]
i could see that working for some, it's not normally how I use it.  i tend to have large playlists and they may play for days and depending on what i am doing sometimes a song just doesn't fit at that moment.  my case could certainly not be the norm though.
